python/tests/server/test_runner.py

```diff
@@ -65,53 +68,57 @@ def test_prediction_runner(runner):
     assert isinstance(response.completed_at, datetime)
 
 
-def test_prediction_runner_called_while_busy(runner):
+@pytest.mark.asyncio
+async def test_prediction_runner_called_while_busy(runner):
     request = PredictionRequest(input={"sleep": 0.1})
     _, async_result = runner.predict(request)
 
     assert runner.is_busy()
     with pytest.raises(RunnerBusyError):
-        runner.predict(request)
+        await runner.predict(request)[1]
 
-    # Call .get() to ensure that the first prediction is scheduled before we
+    # Await to ensure that the first prediction is scheduled before we
     # attempt to shut down the runner.
-    async_result.get()
+    await async_result
 
 
-def test_prediction_runner_called_while_busy_idempotent(runner):
+@pytest.mark.asyncio
+async def test_prediction_runner_called_while_busy_idempotent(runner):
     request = PredictionRequest(id="abcd1234", input={"sleep": 0.1})
 
     runner.predict(request)
     runner.predict(request)
     _, async_result = runner.predict(request)
 
-    response = async_result.get(timeout=1)
+    response = await async_result
```

Comment:
Let's not do away with the timeouts. From experience, the timeouts help make running the tests not an utterly miserable experience if you're working on the code and break it somehow.
```suggestion
    response = await asyncio.wait_for(async_result, timeout=1)
```